packages/core/src/App/Components/Routes/route-with-sub-routes.jsx

```diff
@@ -24,13 +27,11 @@ const RouteWithSubRoutes = route => {
                 to = location.pathname.toLowerCase().replace(route.path, '');
             }
             result = <Redirect to={to} />;
-        } else if (route.is_authenticated && !route.is_logged_in && !route.is_logging_in) {
+        } else if (is_valid_route && route.is_authenticated && !route.is_logged_in && !route.is_logging_in) {
```

Comment:
(throughout) So is this.

Reply:
yes im gonna close this pull request mamali and create another one. Was my mistake